scavenge/tutorial/02-the-game.md

```diff
@@ -4,28 +4,28 @@ order: 2
 
 # The Game
 
- The application we're building today can be used in many different ways but I'll be talking about it as **scavenger hunt** game. Scavenger hunts are all about someone setting up tasks or questions that challenge a participant to find solutions which come with some sort of a prize. The basic mechanics of the game are as follows:
- * **Anyone can post a question with an encrypted answer.**
- * **This question comes paired with a bounty of coins.**
- * **Anyone can post an answer to this question, if they get it right, they receive the bounty of coins.**
+The application we're building today can be used in many different ways but I'll be talking about it as **scavenger hunt** game. Scavenger hunts are all about someone setting up tasks or questions that challenge a participant to find solutions which come with some sort of a prize. The basic mechanics of the game are as follows:
 
+* Anyone can post a question with an encrypted answer.
+* This question comes paired with a bounty of coins.
+*  Anyone can post an answer to this question, if they get it right, they receive the bounty of coins.**
 
- Something to note here is that when dealing with a public network with latency, it is possible that something like a [man-in-the-middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) could take place. Instead of pretending to be one of the parties, an attacker would take the sensitive information from one party and use it for their own benefit. This is actually called [Front Running](https://en.wikipedia.org/wiki/Front_running) and happens as follows: 
- 1. You post the answer to some question with a bounty attached to it.
- 2. Someone else sees you posting the answer and posts it themselves right before you.
- 3. Since they posted the answer first, they receive the reward instead of you.
 
+Something to note here is that when dealing with a public network with latency, it is possible that something like a [man-in-the-middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) could take place. Instead of pretending to be one of the parties, an attacker would take the sensitive information from one party and use it for their own benefit. This is actually called [Front Running](https://en.wikipedia.org/wiki/Front_running) and happens as follows:
```

Comment:
```suggestion
Something to note here is that when dealing with a public network with latency, it is possible that something like a [man-in-the-middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) could take place. Instead of pretending to be one of the parties, an attacker would take the sensitive information from one party and use it for their own benefit. This scenario is actually called [Front Running](https://en.wikipedia.org/wiki/Front_running) and happens as follows:
```